system/settings/win32.xml

```diff
@@ -29,13 +20,6 @@
     </category>
   </section>
   <section id="system">
-    <category id="videoscreen">
-      <group id="1">
-        <setting id="videoscreen.fakefullscreen">
```

Comment:
If I read it correct this negates this setting if windows was running a GL build. So why would it stay if we don't have GL? Or did I misinterpreted the setting?